main.nf

```diff
@@ -35,8 +35,71 @@ include { POSTPROC_ONLY } from './workflows/postproc_only.nf'
 // modules
 include { BUILD_ALLELETABLE } from './modules/local/build_alleletable.nf'
 
+def helpMessage() {
+  log.info """
+    Usage:
+      The typical command for running the pipeline is as follows:
+      nextflow run main.nf --readDIR data/testdata --target v4
+
+    Mandatory arguments:
+      --readDIR    Path to folder containing fastq files
+      --target     The amplicon panel version that was used. [Options: 4cast, ama1, v1, v2, v3, v4 (default)]
+
+    Optional arguments:
+      --outDIR                  Path to folder to place final output
+      --sequencer               The sequencer used to produce your data. [Options: miseq, nextseq (default)]
+      --QC_only                 Whether to only run QC related workflows or all workflows [Options: True (default), False]
```

Comment:
Maybe could also add "Enables QC only workflows when set."? Something with language indicating this is a flag